### DISCUSSION TITLE
Deployment in LXC, version.py and database not working?

### DISCUSSION THREAD

**Discussion by nicedevil007** [TandoorRecipes/recipes#2772] [General]
Heya, I followed the Guide for Installation on a Debian 12 (https://docs.tandoor.dev/install/manual/).

After everything is done I started the service and this is shown to me on the SystemTab.

![image](https://github.com/TandoorRecipes/recipes/assets/17103076/8d2bf756-3563-460f-a5dc-038e08a163a5)

I installed the PostgreSQL Version 15 and also setup the password etc.
This was all put into the `.env` file.

If I run the `version.py` it looks like this:

```bash
root@tandoor:/var/www/recipes# bin/python3 version.py
getting tandoor version
fatal: detected dubious ownership in repository at '/var/www/recipes'
To add an exception for this directory, call:

        git config --global --add safe.directory /var/www/recipes
command '['git', 'show', '-s']' return with error (code 128): b''
writing version info []
root@tandoor:/var/www/recipes# git config --global --add safe.directory /var/www/recipes
root@tandoor:/var/www/recipes# bin/python3 version.py
getting tandoor version
fatal: no tag exactly matches '4723a7ecbd7f5bb77b440a655b73cf8761001764'
writing version info [{'name': 'Tandoor ', 'version': 'commit 4723a7ecbd7f5bb77b440a655b73cf8761001764\nAuthor: vabene1111 \nDate:   Tue Nov 28 20:38:45 2023 +0100\n\n    added pg upgrade faq\n', 'website': 'https://github.com/TandoorRecipes/recipes', 'commit_link': 'https://github.com/TandoorRecipes/recipes/commit/4723a7ecbd7f5bb77b440a655b73cf8761001764', 'ref': '4723a7ecbd7f5bb77b440a655b73cf8761001764', 'branch': 'master\n', 'tag': ''}]
```

Can someone help?

Offtopic: Was anybody able to get this installed on AlmaLinux (or any RHEL based Linux?)

**Comment by smilerz** [comment]
The database warning is wrong. It'll be fixed at the next release.

**Reply by nicedevil007** [reply]
Ah ok, saw that right now version 1.5.10 was released.

Can you also tell me something about the version.py thingi ?
